### ISSUE TITLE
Toolbar moves outside of screen bounds

### ISSUE THREAD

**Issue by sophiamf** [matt-curtis/Fluid-for-Sketch#98]
<img width="963" alt="screen shot 2016-05-19 at 11 16 49 am" src="https://cloud.githubusercontent.com/assets/10751578/15398891/74152804-1db3-11e6-96df-c3361fd02d24.png">
For some reason the toolbar constantly pushes out to off screen when i have multiple files open (i work on multiple monitors) i have to minimize > exposé (screenshot) > drag it over. just very cumbersome. if there was a way to either "reset" the location on the screen or pin the toolbar to the acutal interface (similar to CRAFT) that would be a huge time saver.

**Comment by matt-curtis** [comment]
Wasn't aware of this, it _should_ be pinned to the main window. What version of Sketch & OS X are you running?

**Comment by kozt** [comment]
Same issue here.
Sketch: 39.1
OSX: 10.11.5

Working on a laptop with an external display.

**Comment by matt-curtis** [comment]
@kozt Thanks! Been trying to narrow this one down, that helps.

**Comment by matt-curtis** [comment]
Update: looks like this is being caused by the 'floating window' behavior. Fluid automatically attaches to the current window, and it looks like that's causing issues with multi-monitor setups. This happens when you move between windows on different screens, right?

cc: @joellepeters

**Comment by pulpy** [comment]
Yes, I have this problem too. I use multiple monitors. I have Sketch open on the secondary monitor(extra wide LG 3440x1440). When I try to access the fluid toolbar It does not show up until I move Sketch to the primary monitor (iMac 5k) and access the fluid toolbar a few times then the toolbar shows up.

**Comment by m-reiche** [comment]
Same Problem here with secondary monitor. Sometimes you cannot reach edit constraints window anymore. fixing it by syncing the monitors. positioning of the modals still buggy.
